src/index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet"
+        href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@20..48,100..700,0..1,-50..200" />
+        <title>To-do</title>
+</head>
+
+<body>
+    <main>
+        <header class="head">
+            <h1>Today's To Do</h1>
+            <button>
+                <span class="material-symbols-outlined">
+                    autorenew
+                </span>
+            </button>
+        </header>
+
+        <div class="add-task" action="">
+            <input type="text" name="" id="task" placeholder="Add to your list">
+            <button class="add-btn"> <span class="material-symbols-outlined">
+                    keyboard_return
+                </span></button>
+            </div>
+
+        <div class="tasks"></div>
+        <p class="clear"><a href="#">clear all completed</a></p>
+    </main>
+</body>
```

Comment:
Kindly check on the spaces indentation and make sure it is set to 2 space

Reply:
thanks, I'll fix that 👍